.gitlab-ci.yml

```diff
@@ -67,6 +68,12 @@ lint:
   script:
     - pre-commit run --all
 
+tests:
+  <<:                              *common-refs
+  stage:                           unit_test
```

Comment:
Just ooc, why can't this go into the `test` stage together with the `security-scan` job?

Reply:
I decided to keep them separated just because their concerns are different.